src/components/questions_answers/QnA_Components/AnswerPhotos.jsx

```diff
@@ -1,3 +1,45 @@
-// import React, { useState } from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 
-export default function AnswerPhotos() {}
+export default function AnswerPhotos({ photo }) {
+  const [openModal, setOpenModal] = useState(false);
+
+  const handlePhotoClick = () => {
+    setOpenModal(!openModal);
+  };
+
+  return (
+    <div>
+      <img
+        className="answer-photo"
+        src={photo}
+        alt="product"
+        role="presentation"
+        onClick={handlePhotoClick}
+        onKeyPress={handlePhotoClick}
+      />
+      {openModal ? (
+        <div className="answer-photo-modal ">
+          <button
+            className="enlarged-answer-photo-close"
+            type="button"
+            onClick={handlePhotoClick}
```

Comment:
To make the 'x' button more visible here, would recommend adding the below styling:
`style={{
              position: 'absolute',
              top: '1em',
              right: '1em',
            }}
`